### PR TITLE
add sato_2006_effect_725

### PR DESCRIPTION
**Pull request by linuxrider**

### CODE REVIEW

**Review by saraedum — DISMISSED** [review]

**Comment by linuxrider** [comment]
Due to the unit in the scan rate text this depends on #svgdigitizer/#126.

File: literature/sato_2006_effect_725/sato_2006_effect_725.bib [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+@article{sato_2006_effect_725,
+  title = {Effect of Sulfuric Acid Concentration on the Structure of Sulfate Adlayer on {{Au}}(1~1~1) Electrode},
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725.bib:3]:
```suggestion
  title = {Effect of Sulfuric Acid Concentration on the Structure of Sulfate Adlayer on Au(111) Electrode},
```

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          unit: volume percent
+        source:
+          refinement: Milli-Q SP-TOC
+          quality: ultrapure water
+        purity:
+          resistance:
+            value: 18.2
+            unit: MOhm cm
+      - name: sulfuric acid
+        sum formula: H2SO4
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:27]:
```suggestion
      - name: H2SO4
```

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        type: acid
+        source:
+          supplier: Cica-Merck
+          grade: ultrapur
+          LOT: null
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:32]:
```suggestion
```

Comment (DunklesArchipel) [reply literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:32]: All entries that are empty or contain `null`, `none` etc should be removed

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        concentration:
+          value: 1.0
+          unit: mol L⁻¹
+    pH:
+      estimated value (CurTiPot): 0.1
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:37]: I think this should always be `value`.
Additional information could be stored in a key called `comment'.

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      shape: not given
+    working electrode:
+      material: Au
+      crystallographic orientation: '111'
+      source:
+        supplier: homemade
+      shape:
+        type: bead shaped
+      geometric electrolyte contact area: 
+        value: null
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:59]:
```suggestion
```

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        unit: cm-2
+      preparation procedure: annealed in hydrogen flame (1 min) and subsequently quenched in ultrapure water
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:61]:
```suggestion
      preparation procedure: Annealed in hydrogen flame for 1 min and subsequently quenched in ultrapure water.
```

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  version: 1
+
+curation:
+  version: 1
+  process:
+  - role: curator
+    name: Johannes M. Hermann
+    orcid: https://orcid.org/0000-0001-7119-1295
+    date: 2021-10-11
+
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:75]:
```suggestion
  - role: curator
    name: Johannes M. Hermann
    orcid: https://orcid.org/0000-0001-7119-1295
    date: 2021-10-11
  - role: reviewer
     name: Albert Engstfeld
     orcid: https://orcid.org/0000-0002-9686-3948
     date: 2022-06-22
```

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  instrumentation:
+  - type: potentiostat
+
+system:
+  type: electrochemical
+  electrolyte:
+    type: aq
+    components:
+      - name: water
+        sum formula: H2O
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:14]:
```suggestion
      - name: H2O
```

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
I made some changes to the YAML which would need to be adopted in the other files.
Also, I left you a personal note regarding the digitized curves to make them more smooth.

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+      - name: H2SO4
+        type: acid
+        source:
+          supplier: Cica-Merck
+          grade: ultrapur
+        concentration:
+          value: 1.0
+          unit: mol L⁻¹
+    pH:
+      estimated value (CurTiPot): 0.1
```
[the remark this answers is not in the document]
Comment (linuxrider) [reply literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:34]: @DunklesArchipel Do you agree with this approach?

**Comment by linuxrider** [comment]
I think now all suggestions are applied.

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml (hunk 9 above)
[the remark this answers is not in the document]
Comment (DunklesArchipel) [reply literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:34]: sure.

We should consider it in our metadata-schema. (see https://github.com/echemdb/metadata-schema/issues/22)

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+          supplier: Cica-Merck
+          grade: ultrapur
+        concentration:
+          value: 1.0
+          unit: mol L⁻¹
+    pH:
+      estimation:
+        value: 0.1
+        calculation: CurTiPot
+
```
Comment (DunklesArchipel) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4a_red.yaml:37]:
```suggestion
```

**Review by DunklesArchipel — APPROVED** [review]
Looks good. I removed some empty lines and the instrumentation section, since it did not contain any meaningful information.

**Comment by DunklesArchipel** [comment]
@saraedum there are some outdated change requests, which block the merging process. Could you please approve?

**Comment by linuxrider** [comment]
@saraedum I used the option to dismiss your review of the outdated changes. Hope this is ok for you?
In general, is this bad practice to dismiss a review or is ok under such circumstances?

**Pull request merged**